### PR TITLE
fixed error with the react table keys and the one with the empty header

### PR DESCRIPTION
**Pull request by HrisyToteva**
fixed error:
image

# Additional Review Notes
Put your complementary notes here for helping reviewers review your changes.

### CODE REVIEW

File: src/src/pages/capabilities/MyCapabilities.js [hunk 1]
```diff
@@ -139,15 +140,17 @@ export default function MyCapabilities() {
         }
       },
       {
-        id: 'details',
+        accessorFn: (row) => row.id,
+        header: 'dddd',
```
Comment (kralle333) [inline src/src/pages/capabilities/MyCapabilities.js:144]: dddd?

File: src/src/pages/capabilities/MyCapabilities.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { useContext, useEffect, useMemo, useState } from "react";
 import { Text } from "@dfds-ui/typography";
 import { useNavigate } from "react-router-dom";
 import { ChevronRight, StatusAlert } from "@dfds-ui/icons/system";
+import {Tooltip} from '@mui/material';
```
Comment (kralle333) [inline src/src/pages/capabilities/MyCapabilities.js:5]: are you using this import?

**Review by kralle333 — CHANGES_REQUESTED** [review]

File: src/src/pages/capabilities/MyCapabilities.js (hunk 1 above)
Comment (SEQUOIIA) [reply src/src/pages/capabilities/MyCapabilities.js]: Addressed in new commit

File: src/src/pages/capabilities/MyCapabilities.js (hunk 2 above)
Comment (SEQUOIIA) [reply src/src/pages/capabilities/MyCapabilities.js]: Addressed in new commit

**Review by kralle333 — APPROVED** [review]
looks nice!

**Pull request merged**